mods/ts/rules/palettes.yaml

```diff
@@ -41,6 +41,9 @@
 	PaletteFromFile@effect:
 		Name: effect
 		Filename: anim.pal
+	PaletteFromFile@effectglow:
+		Name: effectglow
```

Comment:
I have two nits/wishes for these palette names:
* It's not really a glow, but a *not* darkening.
* We really should use dashes instead of running words together (I know this issue predates this pr)

What do you think about something like `effect-ignore-lighting`?